smartsim/database/orchestrator.py

```diff
@@ -105,10 +106,15 @@ def _check_run_command(launcher: str, run_command: str) -> None:
         raise SmartSimError(msg)
 
 
-def _get_single_command(run_command: str, batch: bool, single_cmd: bool) -> bool:
+def _get_single_command(
+    run_command: str, launcher: str, batch: bool, single_cmd: bool
+) -> bool:
     if not single_cmd:
         return single_cmd
 
+    if launcher == "dragon":
+        return False
+
     if run_command == "srun" and getenv("SLURM_HET_SIZE") is not None:
```

Comment:
This test doesn't look like it should live in an orchestrator. Perhaps something in a slurm launcher or a slurm commands module could be imported?